### PR TITLE
add_two_number linkedlist.cpp   (#404 )

### PR DESCRIPTION
**Pull request by Akp6528**
Plz mentor @tarun26091999  go through this code

### CODE REVIEW

**Comment by Akp6528** [comment]
@tarun26091999 hello sir, i have made a PR on issue #404 , code written in C++ language plz review and merge it.
Thank u!

**Comment by Akp6528** [comment]
Any mentor plz review this. @raksha009  @rudrakshi99  @tarun26091999

**Comment by Akp6528** [comment]
Any mentor plz review this. @raksha009  @rudrakshi99  @tarun26091999

**Comment by Akp6528** [comment]
Any mentor plz review this. @raksha009 @rudrakshi99 @tarun26091999

File: Code/C++/add_two_numbers_linked_list.cpp [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+/*Input 
+a)
+1    // No. of testcases
+2    // No. of Nodes in List 1
+9 5  // represents number 95
+2    // No. of Nodes in List 2
+8 3  // represents number 83
+Output
```
Comment (rudrakshi99) [inline Code/C++/add_two_numbers_linked_list.cpp:104]: do not use inline comments in test cases, just use `Test case :` in starting and use `input :` and `output :`.  Refer to the code base for format.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Review by Akp6528 — COMMENTED** [review]
Now Check this!!

File: Code/C++/add_two_numbers_linked_list.cpp (hunk 1 above)
Comment (Akp6528) [reply Code/C++/add_two_numbers_linked_list.cpp]: I have done that change @rudrakshi99 
now check it!!!

**Review by rudrakshi99 — APPROVED** [review]

**Review by OjusWiZard — APPROVED** [review]

**Pull request merged**